packages/table/src/headers/columnHeaderCell.tsx

```diff
@@ -98,6 +102,13 @@ export class ColumnHeaderCell extends AbstractComponent<IColumnHeaderCellProps,
             || target.classList.contains(Classes.TABLE_HEADER_CONTENT);
     }
 
+    public constructor() {
+        super();
+        this.state = {
+            isActive: false,
+        };
+    }
```

Comment:
also this constructor implementation is incorrect: must accept `(props, context?)` params. 
but really just remove it.